src/printer.ts

```diff
@@ -626,7 +626,9 @@ export class PugPrinter {
 			case 'newline':
 			case 'outdent':
 			case 'indent': {
-				const result = `${this.computedIndent}${val}`;
+				const computedIndent = this.computedIndent;
+				this.possibleIdPosition = this.result.length + computedIndent.length;
+				const result = `${computedIndent}${val}`;
```

Comment:
When using just `this.computedIndent` the diff will be a bit smaller
Please just use `this.computedIndent` instead of assigning it to a local variable

Reply:
not sure I got the smaller diff part :/

Reply:
See this: https://github.com/prettier/plugin-pug/pull/96/files#diff-46bec2a2249807aa1515b246cc13ee93R629 ?
It's only 1 diff instead of 3 lines